protocols/CountAllFinishedFiles.sh

```diff
@@ -5,17 +5,16 @@
 #string logsDir 
 #string groupname
 
-cd "${projectJobsDir}"
+cd "${projectJobsDir}" || exit
```

Comment:
Why is this necessary? If the `cd` fails the script will exit due to `set -e` anyway, right?

Reply:
The set -e is part of the header, so not visible here; maybe the reason shellcheck got confused?